insta-clone/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import CommentsSection from '../CommentsSection/CommentsSection';
+import './PostContainer.css';
+import PropTypes from 'prop-types';
+
+
```

Comment:
Very minor, but a linter may say there's an error if there is more than 1 line-space between sections.

Reply:
Ok, got it fixed